code/controllers/subsystem/atoms.dm

```diff
@@ -173,9 +173,8 @@ SUBSYSTEM_DEF(atoms)
 /datum/controller/subsystem/atoms/proc/setupGenetics()
 	var/list/mutations = subtypesof(/datum/mutation/human)
 	shuffle_inplace(mutations)
-	for(var/A in subtypesof(/datum/generecipe))
-		var/datum/generecipe/GR = A
-		GLOB.mutation_recipes[initial(GR.required)] = initial(GR.result)
+	for(var/datum/generecipe/GR as anything in subtypesof(/datum/generecipe))
+		GLOB.mutation_recipes |= GR
```

Comment:
Is there any point in keeping `GLOB.mutation_recipes` when it's literally just `subtypesof(/datum/generecipe)`?

Reply:
is there any performance gain over storing them vs using subtypesof? I'm not familiar with internals but if there isn't then I can just kill mutation_recipes

Reply:
Unless `get_mixed_mutation` is called like 100 times a second then there's no performance concerns really